hedera-node/src/test/java/com/hedera/services/ledger/BaseHederaLedgerTest.java

```diff
@@ -99,6 +101,7 @@
 	protected AccountID rand = AccountID.newBuilder().setAccountNum(2_345).build();
 	protected AccountID deleted = AccountID.newBuilder().setAccountNum(3_456).build();
 	protected AccountID detached = AccountID.newBuilder().setAccountNum(4_567).build();
+	protected FCOneToManyRelation<EntityId, MerkleUniqueTokenId> uniqueTokenAccountOwnerships;
```

Comment:
Is this change necessary?

Reply:
some unit tests were failing with null pointers